app/src/main/java/de/danoeh/antennapod/fragment/swipeactions/SwipeActions.java

```diff
@@ -0,0 +1,298 @@
+package de.danoeh.antennapod.fragment.swipeactions;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.graphics.Canvas;
+
+import androidx.annotation.NonNull;
+import androidx.core.graphics.ColorUtils;
+import androidx.core.util.Pair;
+import androidx.fragment.app.Fragment;
+import androidx.lifecycle.Lifecycle;
+import androidx.lifecycle.LifecycleEventObserver;
+import androidx.recyclerview.widget.ItemTouchHelper;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.annimon.stream.Optional;
+import com.annimon.stream.Stream;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import de.danoeh.antennapod.R;
+import de.danoeh.antennapod.dialog.SwipeActionsDialog;
+import de.danoeh.antennapod.fragment.EpisodesFragment;
+import de.danoeh.antennapod.fragment.QueueFragment;
+import de.danoeh.antennapod.model.feed.FeedItem;
+import de.danoeh.antennapod.model.feed.FeedItemFilter;
+import de.danoeh.antennapod.ui.common.ThemeUtils;
+import de.danoeh.antennapod.view.viewholder.EpisodeItemViewHolder;
+import it.xabaras.android.recyclerview.swipedecorator.RecyclerViewSwipeDecorator;
+
+public class SwipeActions {
+    public static final String PREF_NAME = "SwipeActionsPrefs";
+    public static final String PREF_SWIPEACTIONS = "PrefSwipeActions";
```

Comment:
I was just confused about the naming here, too. One is the key and one is the preferences name, right? So I would call the second one something like `KEY_SWIPEACTION_PREFIX`

Reply:
I just copied the convention of the other Fragments, feel free to change though